modules/encrypted_logs/README.md

```diff
@@ -0,0 +1,30 @@
+# Encrypted Logs
+
+This module provisions a CloudWatch log group encrypted with KMS.
+
+## Usage
+
+```terraform
+module "encrypted_logs" {
+  log_group_name = "encrypted-logs"
+  tags = {
+    environment = "staging"
```

Comment:
Might have a mistake here? Shouldn't the _righthand_ side be `var.log_group_name` instead of being a static string?

Reply:
Oh, this is actually specifying what the value for `var.log_group_name` is when you call the module. However, I did realize I didn't actually put in the `source` part to make it clear that this is how you would call the module! Amending that now.

Reply:
Right! That's what I get for mistaking the README for main.tf. 🤦